index.html

```diff
@@ -26,24 +24,259 @@ <h1>Hey there. I'm Richard</h1>
+      </section>
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+
+   </div>
+
+   <div class="about-me">
```

Comment:
- [ ] The container of the section should a `section` tag. Please change it for good practice. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.).